### PR TITLE
Add examples for `mesheryctl` commands

### LINKED ISSUE #3978: [Docs] Refactor examples of mesheryctl CLI commands/subcommands
#### Current State
There are some mesheryctl CLI commands/subcommands that are having placeholder values.
e.g. https://docs.meshery.io/reference/mesheryctl/mesh
c4
c3

https://docs.meshery.io/reference/mesheryctl

#### Desired State
The examples could be actual working examples instead of using placeholder values. Add such examples for every subcommand
https://github.com/meshery/meshery/pull/3841#pullrequestreview-733580138

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
**Description**

This PR fixes #3978 

**Notes for Reviewers**
Quick check list...
- [x] global
- [x] system (some commands)
- [x] system-context
- [x] system-channel
- [x] perf 
- [x] mesh
- [x] pattern
- [x] app

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 33acf001de3991290b21355cb1467e2b2c13285b at: https://6153fa192e049b8efa8d102d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 67bea2edf0ebb2d8bbd0918118ace4d512a292fd at: https://615408c266f6dd956f62792e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3a3eebd6e062ebda9079b17a4c05e6f877e46ad0 at: https://615475d06e6a851358226c9b--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
@navendu-pottekkat could you tell what are the file formats for the SMI conformance file and patterns? For some commands, it's needed and I'm clueless about them. Plus, even the patterns too are bit hard to understand at first glance, like a novice user finds it difficult to implement.

**Comment by leecalcote** [comment]
@alphaX86 some good stuff happening here!

**Comment by alphaX86** [comment]
> could you tell what are the file formats for the SMI conformance and patterns? For some commands, it's needed and I'm clueless about them. Plus, even the patterns too are bit hard to understand at first glance, like a novice user finds it difficult to implement.

@leecalcote could you answer my query so that I can move on?

//cc: @sayantan1413 (since you worked with patterns)

**Comment by leecalcote** [comment]
@alphaX86 great question. We should not only add examples to `mesheryctl` on these, but also in the Pattern Management and other pages.

- patterns are of the file format yaml. Here are sample patterns: https://github.com/service-mesh-patterns/service-mesh-patterns. Each of these must contain a `name` property and should contain one or more `services`.
- applications are of the file format yaml.
- filters are of a binary file format.

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 1]
```diff
@@ -1,26 +1,35 @@
 global:
   name: mesheryctl
-  description : Displays help about any command.
+  description : Global command
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:3]:
```suggestion
  description : A global command that displays an overview of all commands.
```

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
   usage:
     mesheryctl
+  example: |
+    mesheryctl system start
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:7]: This isn't an example of the global command...

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
+      mesheryctl --help
+      mesheryctl system stop
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:9]: This isn't an example of the global command...

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 4]
```diff
@@ -1,26 +1,35 @@
...
 
   flags:
     config:
       name: --config
       description: configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE/EKS).
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:14]: This description is incorrect. This is the meshconfig.

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 5]
```diff
@@ -43,6 +52,8 @@ system:
       description: Temporarily change the current context.
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:52]: Please elaborate on the concept of contexts.

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 6]
```diff
@@ -43,6 +52,8 @@ system:
...
       usage:
         mesheryctl system reset -c [temporary context name]
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:54]: Please provide explanation of what this example usage actually does.

**Comment by leecalcote** [comment]
https://github.com/meshery/meshery/pull/4269/files#diff-da82e4d8bc231e812de092c80bcf1c565619d07b14d2852ddb730396e16f84dbR38 the description is inaccurate.

this command confirms the version number of meshery client installed on the user's system and the meshery server version of their current context. It also verifies whether the user is running the latest release version or not.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 43b6b1e69349b0bf4ed60aa8533a9c129efc95d9 at: https://61594c350e0b4899c3be48b0--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit dd123087dff8e33d361de54481c8b21763fb4b04 at: https://6159508319ab3e9600fde837--meshery-docs-preview.netlify.app

**Review by warunicorn19 — DISMISSED** [review]
Till now the changes LGTM, Good stuff! @alphaX86

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 7]
```diff
@@ -98,18 +112,27 @@ system:
       description: Output shell completion code.
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:112]:
```suggestion
      description: Output shell completion code for the specified shell.
```

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 8]
```diff
@@ -130,14 +153,11 @@ system:
     logs:
       name: logs
       description: Starts tailing Meshery server debug logs
-      usage: |
-
-          # View the logs for a specific component
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:135]: This update loses the fact that users can filter logs for a specified component.

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 9]
```diff
@@ -187,13 +206,19 @@ system-channel:
       description: set release channel and version.
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:206]:
```suggestion
      description: Subscribe to release channel and version.
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
We're almost there!

**Comment by l5io** [comment]
🚀 Preview for commit 30693f5e52d5235c58bdfd668dfbbca2715b88d1 at: https://615e661c75213464fb0219e1--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
And BTW @navendu-pottekkat you didn't answer this question yet. What's the file format for a SMP compatible configuration file? I need to use it for `perf --file` I tired but only GUI approach I could do and for CLI approach I'm confused for that alone

**Comment by navendu-pottekkat** [comment]
The other comment that you only see it in the main page alone. Cool. No problem.

Please ping when you make the edit in `perf` like you mentioned and we can merge this PR/

**Comment by alphaX86** [comment]
@navendu-pottekkat it's done...

@leecalcote help here! DCO again! :')

**Comment by leecalcote** [comment]
yay!

**Pull request merged**